### PR TITLE
Fix `pc log -h` listed active loggers out of date

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
The activeLoggers slice has not been maintained for a while, and it's hard to maintain manually. Many logger components are missing. Revised to use command or see the logger page like Envoy doc does in the tip (https://www.envoyproxy.io/docs/envoy/latest/start/quick-start/run-envoy#debugging-envoy). Are there any other suggestions?

### CODE REVIEW

File: istioctl/cmd/proxyconfig.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ const (
 	prometheusMergedOutput = "prom-merged"
 
 	defaultProxyAdminPort = 15000
+	loggerIDs             = "https://github.com/envoyproxy/envoy/blob/a62fa454ffa4c28bec01d37788a38e71a0e230d3//source/common/common/logger.h#L29"
```
Comment (ymesika) [inline istioctl/cmd/proxyconfig.go:46]: Is this the correct line? Should it be L35?

Comment (ymesika) [reply istioctl/cmd/proxyconfig.go:46]: Btw, where is this being used?

File: istioctl/cmd/proxyconfig.go [hunk 2]
```diff
@@ -795,14 +751,13 @@ func logCmd() *cobra.Command {
...
-	s := strings.Join(activeLoggers, ", ")
 
 	logCmd.PersistentFlags().BoolVarP(&reset, "reset", "r", reset, "Reset levels to default value (warning).")
 	logCmd.PersistentFlags().StringVarP(&labelSelector, "selector", "l", "", "Label selector")
 	logCmd.PersistentFlags().StringVar(&loggerLevelString, "level", loggerLevelString,
 		fmt.Sprintf("Comma-separated minimum per-logger level of messages to output, in the form of"+
-			" [<logger>:]<level>,[<logger>:]<level>,... where logger can be one of %s and level can be one of %s",
-			s, levelListString))
+			" [<logger>:]<level>,[<logger>:]<level>,... where logger components can be listed by running \"istioctl proxy-config log <pod-name[.namespace]>\""+
+			"or referred from https://github.com/envoyproxy/envoy/blob/main/source/common/common/logger.h, and level can be one of %s", levelListString))
```
Comment (ymesika) [inline istioctl/cmd/proxyconfig.go:760]: Did you plan to use the constant defined above here?

File: istioctl/cmd/proxyconfig.go (hunk 1 above)
Comment (hanxiaop) [reply istioctl/cmd/proxyconfig.go]: Forgot deleting this, thanks

**Comment by hanxiaop** [comment]
not stale

**Review by GregHanson — APPROVED** [review]

**Pull request merged**